lib/osf-components/addon/components/contributors/card/readonly/component.ts

```diff
@@ -1,11 +1,38 @@
 import { tagName } from '@ember-decorators/component';
 import Component from '@ember/component';
 
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 import { layout } from 'ember-osf-web/decorators/component';
+import ContributorModel from 'ember-osf-web/models/contributor';
 import styles from './styles';
 import template from './template';
 
 @layout(template, styles)
 @tagName('')
 export default class ContributorsCardReadonly extends Component {
+    // required
+    contributor!: ContributorModel;
+
+    // private properties
+    employmentList!: string[];
+    educationList!: string[];
+
+    @tracked showDropdown = false;
+
+    didReceiveAttrs() {
+        const employment = this.contributor.users.get('employment') || [];
+        const education = this.contributor.users.get('education') || [];
+        this.employmentList = Array.from(
+            employment, item => [item.institution, item.department].join(', ').replace(/,\s*$/, ''),
```

Comment:
Drive-by: This seems like it should be template work, rather than javascript work. Why not just pass the `employment` and `education` into the template and pull out what you need from there? That would make it easier in the future if we ever need to do something different with the display of education and employment information.